README.md

```diff
@@ -1,79 +1,103 @@
-# Tera - An Internet-Scale Database
+# Tera - A distributed high performance NoSQL database
 
 [![Build Status](http://220.181.7.231/buildStatus/icon?job=tera_master_build)](http://220.181.7.231/job/tera_master_build/)  
 Copyright 2015, Baidu, Inc.
 
-Tera is a structured distributed database which is designed to manage trillions of links and webpages. Besides Webpage Database, Tera is used by many other Baidu products, including Baidu Trace System, Baidu User Behavior Analysis System, etc. Tera provides high performance and high scalability by taking fully advantage of new generation hardware.
+Tera is a distributed high performance NoSql database, which is inspired by google's [BigTable](http://static.googleusercontent.com/media/research.google.com/zh-CN//archive/bigtable-osdi06.pdf) and can manage __petabytes__ of data across __thousands__ of commodity servers. Tera is widely used in many Baidu products, including web indexing, WebPage DB, LinkBase DB, etc. 
 
-# Features
+## Features
+
+* Linear and modular scalability
 * Automatic and configurable sharding
 * Ranged and hashed sharding strategies
-* Strong consistency
+* MVCC
+* Column-oriented storage and locality group support
+* Strictly consistent
+* Automatic failover support
 * Online schema change
-* Snapshot
-* Multi-level cache and Bloom Filters to suport real-time queries
+* Snapshot support
+* Support RAMDISK/SSD/DFS tiered cache
+* Block cache and Bloom Filters for real-time queries
+* Multi-type table support (RAMDISK/SSD/DISK table)
+* Easy to use [C++](doc/sdk_dev_guide.md)/[Java](sdk_dev_guide_for_java.md)/[python](sdk_dev_guide_for_python.md)/[REST-ful](doc/http_proxy.md) API for client access
+
+## Data Model
 
-# Contact Us
-{dist-lab, tera_dev, opensearch} at baidu.com
+As a NoSQL database, tera is the collection of many sparse distributed, persistent multidimensional tables. The table is indexed by a row key, column key, and a timestamp; each value in the table is an uninterpreted array of bytes.
 
+* (row:string, (column family+qualifier):string, time:int64) → string
 
-[高性能、可伸缩的结构化数据库](http://github.com/baidu/tera)
-====
+To know more about the schema, you can refer to [BigTable](http://static.googleusercontent.com/media/research.google.com/zh-CN//archive/bigtable-osdi06.pdf).
```

Comment:
To know（know more -> learn more） more about the schema, you can refer to [BigTable]